ADOPTERS.md

```diff
@@ -107,3 +107,4 @@ _If you're using Backstage in your organization, please try to add your company
 | [VIA](https://www.via.com.br)                                               | [@vagnerguedes](https://github.com/vagnerguedes)                                                                                                                                                                                                                 | Centralized Developer Experience portal - Software catalog and documentation platform, software templates, techdocs, scaffolding, self-service infrastructure                                                                                                                                                                                        |
 | [Surevine](https://www.surevine.com/)                                       | [@DJDANNY123](https://github.com/djdanny123)                                                                                                                                                                                                                     | Developer portal for software catalog, discovery and a view of the technologies we are using across the organisation, we are looking to explore how we can enrich our entities in Backstage by integrating a software bill of materials.                                                                                                             |
 | [Bonial International GmbH](https://www.bonial.com/)                        | [@pjungermann](https://github.com/pjungermann)                                                                                                                                                                                                                   | Centralized developer portal with software catalog, tech docs, templates, and more.                                                                                                                                                                                                                                                                  |
+| [Beez Innovation Labs Pvt. Ltd](https://www.beezlabs.com.com/) | [Karthikeyan Venkatesan](https://github.com/karthikeyan23) | Developer portal with software catalog, scaffolding, tech docs, templates, and infra. |
```

Comment:
It looks like there snuck in an additional `.com` in the url :)

Reply:
Fix'd x)

Reply:
Oh yes!

Reply:
Thanks a lot @jhaals @Rugvip 😉